ls1intum/Artemis
# [Exam] Make grace period editable and display additional attributes on participation start cover

### Checklist

### Motivation and Context
The grace period should be editable and we want to provide the students with additional information about the exam.

### Description
- [x] I've added the attributes `examiner`, `moduleNumber` and `courseName`
- [x] I've displayed the attributes and more on the participation start cover (see last screenshot)
- [x] I've made the grace period editable

### Steps for Testing
1. Log in to Artemis
2. Create a new exam with all three new attributes and with an individual grace period
3. Check that the attributes are saved
4. Edit the same attributes
5. Check that the attributes were updated
6. Participate in the exam: check that everything is displayed on the participation start cover (compare with the last screenshot)

### Screenshots
![image](https://user-images.githubusercontent.com/15168372/87595325-a91d3080-c6ee-11ea-901c-bc6ee3bc68d9.png)
![image](https://user-images.githubusercontent.com/15168372/87595358-b1756b80-c6ee-11ea-8c41-28d440397010.png)
![image](https://user-images.githubusercontent.com/15168372/87595364-b4705c00-c6ee-11ea-955b-6522028e0513.png)
